jbmc/src/java_bytecode/README.md

```diff
@@ -883,4 +883,27 @@ To be documented.
 
 \subsection java-trace-string-assignments String assignments
 
-To be documented.
+"Strings" only exist in Java. This section refers to the `java.lang.String`
```

Comment:
How about "String literals are a Java source language concept", since Strings of various sorts clearly do exist in Java and also other languages